### PR TITLE
Fix typo and made commands uniform

### PR DESCRIPTION
**Pull request by Tanmoytkd**

### CODE REVIEW

File: content/docs/tutorials/local.md [hunk 1]
```diff
@@ -183,7 +183,7 @@ In addition, Vitess requires the software and libraries listed below.
 
     ```sh
     # Remaining commands to build Vitess
-    . ./dev.env
+    source ./dev.env
```
Comment (deepthi) [inline content/docs/tutorials/local.md]: Welcome! Can you fix the commit comment?

Comment (Tanmoytkd) [reply content/docs/tutorials/local.md]: I can't see anything wrong with the comments. Can you please elaborate on where the problem is? Which one should I fix? 

"fixed a typo in teardown command"

or 

replaced ". ./dev.env" with "source source.env" for uniformity?

Comment (deepthi) [reply content/docs/tutorials/local.md]: It should be `replaced ". ./dev.env" with "source **dev**.env" for uniformity`

Comment (Tanmoytkd) [reply content/docs/tutorials/local.md]: Thanks a lot for pointing out my mistake. I fixed the commit message.

**Pull request merged**